CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+# 3.0.0-rc.4 (2018-11-30)
+
+- Updated dependency to properly style the hover state for a selected, non-disabled tab. [blackbaud/skyux-theme#41](https://github.com/blackbaud/skyux-theme/pull/41)
```

Comment:
This is minor, but I think "for selected, non-disabled tab" is awkward and difficult to process, especially "non-disabled," which reads like a double negative. Could we change this to "for a selected tab that is active"? ... Or if changing "non-disabled" to "active" doesn't work, then how about "... for a selected tab that is not disabled"?

Anyway, please just let me know if neither of these suggestions work, and I can play around with it some more. Other than that awkward phrasing, everything looks good to me here. Thanks!

Reply:
Yeah I agree. @Blackbaud-TrevorBurch what do you think about "for a selected tab that is active" ?

Reply:
I went with "... for a selected tab that is not disabled"

"Active" could mean one that is clicked on so I didn't want to go with that. Let me know if you have any other suggestions.